lib/active_model/serializable_resource.rb

```diff
@@ -18,6 +18,7 @@ def initialize(resource, options = {})
     # returns the contents of the block
     def self.serialize(resource, options = {})
       serializable_resource = SerializableResource.new(resource, options)
+      serializable_resource.notify_active_support
       if block_given?
         yield serializable_resource
       else
```

Comment:
Thanks for the review @bf4 :heart: 

About the event name I think it is right, I followed the convention `action.class_name` is the same that is used in the Active Job "perform_start.active_job" and in the [Rails guides](http://edgeguides.rubyonrails.org/active_support_instrumentation.html).

About the logger I think we can expose this in a logger too, but in the moment I am only using the instrumentation to make to use in the tests. But yes, we can do this, I think this belongs to other PR and not this. What you think?